### ISSUE TITLE
With cheapglk, transcript created in ISO8859 instead of utf-8 despite `-u`

### ISSUE THREAD

**Issue by stormi** [erkyrath/glulxe#38]
This may be another obvious question, but I'm using glulxe+cheapglk+RegTest, and all is well, however when I tried to simply my test writing by starting with a transcript produced by `glulxe -u mygame.ulx`, I discovered that despite the console output is utf8, the produced transcript is encoded in ISO5589.

Is it the expected behaviour?

**Comment by erkyrath** [comment]
This is a problem with the library/kit code, which calls glk_stream_open_file() rather than glk_stream_open_file_uni().

I thought this was planned to be updated in the next Inform release, but I'm not sure. I'll check.

**Comment by stormi** [comment]
It's possible I'm using too old a version of the compiler or the libs.

**Comment by erkyrath** [comment]
No, the current versions still use byte mode for transcripts (which means ISO8859).

I filed https://inform7.atlassian.net/jira/software/c/projects/I7/issues/I7-2516 .
